app/models/user.js

```diff
@@ -40,5 +43,9 @@ export default Addressable.extend({
 
   fullName: function(){
     return (this.get('firstName') + " " + this.get('lastName'));
-  }.property('firstName', 'lastName')
+  }.property('firstName', 'lastName'),
+
+  lName: function(){
```

Comment:
@steveyken Here for default_user, we have used first_name as "Goodcity Team" and no last_name value. Due to this wherever we were displaying name of user using translation, it was displaying "Goodcity Team null". 
To fix this we added lName property.